src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2879,6 +2879,18 @@ components:
         secureMonitoringEnabled:
           type: boolean
           default: false
+        cloudPlatform:
+          $ref: '#/components/schemas/CloudPlatform'
+        dataProjects:
```

Comment:
When do we end up with multiple data projects per dataset? Is this accounting for when we have multiple datasets per snapshot?

Reply:
It's when we ingest with different billing profiles

Reply:
Ok - I don't think that's being queried for in these changes.  When we ingest with a different billing profile, that does not change the google project associated with the dataset.  Instead, we associate the new google project in the bucket_resource table which is linked to the dataset via the dataset_bucket table.

Reply:
This can happen if someone ingests using two different billing projects.

Reply:
Would we want to instead keep in plural and also query for the bucket projects? Not sure if that would be confusing to the user.